docs/types/react-swipeable-views-utils.d.ts

```diff
@@ -0,0 +1,23 @@
+// TODO: upstream
```

Comment:
Preparing. Hold PR for upstream merge.

Reply:
Thanks for that, I hope I will find a new maintainer for the project :).

Reply:
These guys have built some serious automation tools 🎉. It seems they publish right after pull requests are merged. Pretty cool. Sounds like this marks the completion of the migration of the demos to TypeScript 👌
https://www.npmjs.com/package/@types/react-swipeable-views-utils